app/_locales/en/messages.json

```diff
@@ -4745,6 +4745,50 @@
   "publicAddress": {
     "message": "Public address"
   },
+  "pushNotificationLimitOrderFilledDescriptionLong": {
+    "message": "Your $1 long position is now open.",
+    "description": "$1 is the asset symbol"
+  },
+  "pushNotificationLimitOrderFilledDescriptionShort": {
+    "message": "Your $1 short position is now open.",
+    "description": "$1 is the asset symbol"
+  },
```

Comment:
2 different descriptions to hopefully make it more reliable when creating other translations.
Otherwise the description would look like:

```
"Your $1 $2 position is now open.",
```

This would be much more confusing to translate.